larq/math_test.py

```diff
@@ -0,0 +1,16 @@
+import tensorflow as tf
+import numpy as np
+import larq as lq
+import pytest
+
+
+@pytest.mark.skipif(not tf.executing_eagerly(), reason="requires eager execution")
+def test_sign():
+    # test random
+    x = np.random.rand(100) * 100 - 0.5
+    y = lq.math.sign(x)
+    np.testing.assert_allclose(y.numpy(), np.array(x >= 0, dtype=np.float32) * 2 - 1)
```

Comment:
If you use `tf.keras.backend.get_value(y)`, you can make this test run for both eager and legacy mode.